apps/web/src/common/components/Chips/config.tsx

```diff
@@ -33,6 +33,13 @@ export const genEdChipConfig: GenEdChipConfigProps = {
   NO: { label: 'ไม่ใช่ Gened', color: 'deepGrayOutlined', variant: 'outlined' },
 }
 
+export type GradingChipKey = GradingType
+export type GradingChipConfigProps = Record<GradingChipKey, ChipConfigProps>
+export const gradingChipConfig: GradingChipConfigProps = {
+  S_U: { label: 'S/U Grade', color: 'blueOutlined', variant: 'outlined' },
```

Comment:
Use `S/U` to be consistent with course detail page
```suggestion
  S_U: { label: 'S/U', color: 'blueOutlined', variant: 'outlined' },
```

Reply:
I decided to use S/U Grade in Course Detail Page instead so it align with "Letter Grade"

Reply:
<img width="1038" alt="image" src="https://github.com/thinc-org/cugetreg/assets/24814968/97d31de9-60d2-4cb4-bbbe-0a634f6c5796">
It's still S/U here na

Reply:
Yes, will update in this PR